functions/HttpTriggerAPIUsersId/__init__.py

```diff
@@ -58,8 +85,107 @@ def get_user_by_id(conn, id):
             columns = [column[0] for column in cursor.description]
             data = dict(zip(columns, row))
 
+            logging.debug("Users retrieved successfully!")
             return func.HttpResponse(
                 json.dumps(data),
                 status_code=200,
                 mimetype="application/json"
             )
+
+
+def post():
+    logging.debug("POST is not implemented")
+    return func.HttpResponse(
+        "Method not allowed!",
+        status_code=405
+    )
+
+
+def put(req, conn, id):
+    user_req_body = req.get_json()
+    logging.debug("Verifying fields in request body to update a user by ID")
+    try:
```

Comment:
Have we decided that they have to have all the fields in their request even if they are only updating one of  them?

Reply:
This would be a patch request if you want to update something specific. in this case for Put I want to update all fields

Reply:
Jenni, just an FYI here, if the client wants to update firstName only here via patch, and not supply lastName and email, they will get an error/so one could argue this is not a true patch